### PR TITLE
Updated French & Spanish translations

### PR DESCRIPTION
**Pull request by jo-spek**
Few new strings added to french strings.xml.
Spanish strings.xml completely updated to match English strings.xml.
Spanish translations kindly provided by PilarValbuena <pilar.valbuena.perez@gmail.com >

### CODE REVIEW

File: ground/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -23,48 +23,172 @@
   -->
 
   <!-- Shown below fields that are required but left empty (TBD). -->
-  <string name="required_task">Este campo es obligatorio</string>
+  <string name="required_task">This field is required</string>
```
Comment (shobhitagarwal1612) [inline ground/src/main/res/values-es/strings.xml:26]: Is this change intentional?

**Review by shobhitagarwal1612 — CHANGES_REQUESTED** [review]

File: ground/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (jo-spek) [reply ground/src/main/res/values-es/strings.xml:26]: Nope, that's a mistake!

Comment (jo-spek) [reply ground/src/main/res/values-es/strings.xml]: Fixed

**Review by shobhitagarwal1612 — APPROVED** [review]
